lisa/operating_system.py

```diff
@@ -332,6 +352,37 @@ def _initialize_package_installation(self) -> None:
         # sub os can override it, but it's optional
         pass
 
+    def _get_package_information(self, package_name: str) -> VersionInfo:
+        raise NotImplementedError()
+
+    def _get_version_info_from_named_regex_match(
+        self, package_name: str, named_matches: re.Match[str]
+    ) -> VersionInfo:
+
+        assert_that(named_matches.group("package_name")).described_as(
+            "While capturing package version info, found name of package was "
+            "unexpected. Version info may be incorrect or regex may need updating."
+        ).is_equal_to(package_name)
+        patch_match = named_matches.group("patch")
+        if patch_match:
+            patch_match = "0"
+
+        major, minor, patch = map(
+            int,
+            [named_matches.group("major"), named_matches.group("minor"), patch_match],
+        )
+        build_match = named_matches.group("build")
+        self._node.log.info(
```

Comment:
1. make it debug level.
2. The information can print out the original parsed information, because it needs the exact raw string sometimes.